include/EnvelopeAndLfoParameters.h

```diff
@@ -167,7 +168,7 @@ public slots:
 	sample_t * m_lfoShapeData;
 	sample_t m_random;
 	bool m_bad_lfoShapeData;
-	SampleBuffer m_userWave;
+	std::shared_ptr<const SampleBuffer> m_userWave = std::make_shared<SampleBuffer>();
```

Comment:
A number of classes default to an empty sample buffer like this. Given that they're shareable anyway, I wonder if it would be more efficient to have a single static empty buffer to use here (e.g. `SampleBuffer::empty()`).